### DISCUSSION TITLE
AI for August 🚀  [Repositories Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#137199] [Repositories]
![Header for the Repositories category shows a dark purple abstract background with white text reading “Repositories Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a group of repositories. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/06f2261b-e8f4-40a3-b92b-99cf29b86134)

As we round out August we want to make sure you have the latest and greatest community member shoutouts, what releases have shipped, and content to help you level up! 

## Wall of Fame 🤩 : These folks make our category so great !

* @romhenri: Love how their answer shared different recommendations based on the original poster’s GitHub skill level in [this discussion about repository creation](https://github.com/orgs/community/discussions/134770). 
* @TiagoR07: [self-solved](https://github.com/orgs/community/discussions/135994#discussioncomment-10364952) their download from LFS! Way to troubleshoot and as always thank you for coming back to share the answer 🙇. 
* @DRNadler: [Asked a great question](https://github.com/orgs/community/discussions/135244) on resolving merge conflicts on a forked repository; they were specific in their needs and shared plenty of details. They’re still looking for a solution if you want to go be their hero - [answer the discussion now](https://github.com/orgs/community/discussions/135244). 

## 🗳️ Repository Releases: All updates, info, releases, etc that are Repo-related!

* **New Enhanced Repo Insights Views:** We’re thrilled to introduce improvements to Repo Insights with significant improvements to two insight views. [Give feedback in our Discussion](https://github.com/orgs/community/discussions/135572). 
* **Repository Updates July 31st**: We’re excited to bring an updated repository list view experience and the ruleset merge queue rule to general availability, as well as an update to the status check and workflow rules. [Read through the details on our Changelog](https://github.blog/changelog/2024-07-31-repository-updates-july-31st-2024/). 
* **Private Mirrors App beta (open source):** We've been working on an open source GitHub App that manages private mirrors of public upstream repositories. Due to the nature of the GitHub fork network, this isn't (currently!) possible with a built-in workflow, so the app will handle the creation, synchronization, and deletion of these private mirrors as a standalone experience. [Learn more and give feedback](https://github.com/orgs/community/discussions/133150). 

## 📚 Rousing Reads and Beyond: GitHub-produced goodies 🍭

* **Survey - The AI wave continues to grow on software development teams:** We surveyed 2,000 people on software development teams at enterprises in the U.S., Brazil, India, and Germany about the use, experience, and expectations around generative AI tools in software development. [Read the blogpost](https://github.blog/news-insights/research/survey-ai-wave-grows/) by @kdaigle and our team to see what we learned! 
* **How to add code to your repository:** Getting started on GitHub means understanding the GitHub flow and how to use it to add code to our repository. @ladykerr covers using GitHub to share resources with others and work collaboratively in her [YouTube video](https://www.youtube.com/watch?v=g2XjJhrGGg4).

## 📣 **Announcements**: The latest updates in the Community 🆕

* **Introducing the Models category:** We are enabling the rise of the AI engineer with GitHub Models–bringing the power of industry leading large and small language models to our more than 100 million users directly on GitHub. We invite you to be among the first to experience GitHub Models by signing up for the [waitlist](https://github.com/marketplace/models). [Check out the category](https://github.com/orgs/community/discussions/categories/models) to learn more and see how folks are using the models. 

How are you using AI in your work or personal projects? Share below!

**Comment by 23Benji** [comment]
I use AI to assist with coding projects, like optimizing Python applications, debugging C code, and brainstorming ideas for new features. It also helps with language processing and maintaining context in ongoing conversations.

**Reply by queenofcorgis** [reply]
Thanks for sharing @23Benji - any helpful prompts you have found when brainstorming that you'd like to share?

**Comment by queenofcorgis** [comment]
We're looking back on the great work in September, in [our Discussion with the latest check-in](https://github.com/orgs/community/discussions/139750), come join us!
